roles/forklift_versions/molecule/default/verify.yml

```diff
@@ -59,27 +45,27 @@
       vars:
         scenario: "{{ pipeline_type }}"
         scenario_os: "{{ pipeline_os }}"
-        scenario_version: "4.9"
+        scenario_version: "4.13"
         upgrade: True
     - name: Ensure upgrade steps have been determined correctly
       assert:
         that:
-          - forklift_upgrade_version_start == '4.7'
-          - forklift_upgrade_version_intermediate == '4.8'
-          - forklift_upgrade_version_final == '4.9'
+          - forklift_upgrade_version_start == '4.12'
+          - forklift_upgrade_version_intermediate == '4.12'
+          - forklift_upgrade_version_final == '4.13'
 
     - name: "Include forklift_versions for upgrades with upgrade_step=2"
       include_role:
         name: "forklift_versions"
       vars:
         scenario: "{{ pipeline_type }}"
-        scenario_os: "{{ pipeline_os }}"
-        scenario_version: "4.9"
+        scenario_os: "almalinux8"
+        scenario_version: "4.13"
         upgrade: True
         upgrade_step: 2
     - name: Ensure upgrade steps have been determined correctly
       assert:
         that:
-          - forklift_upgrade_version_start == '4.5'
-          - forklift_upgrade_version_intermediate == '4.7'
-          - forklift_upgrade_version_final == '4.9'
+          - forklift_upgrade_version_start == '4.9'
```

Comment:
@evgeni how am i suppose to manage `4.9` version here when we no longer support this?

Reply:
i believe the failure is coming form here

Reply:
You are correct, this fails now because you deleted 4.9. The way this test works (Upgrade from N-4 to N-2 to N), it means we need to keep N-4 in versions.yaml, even if it's unsupported by us anymore. Given there is 4.14 already, we could update `scenario_version` to 4.14 and then this test uses 4.10 - 4.12 - 4.14, which is enough to delete 4.9.
But then we have to keep 4.10 -- but I think you intended that anyway based on the title of this PR?